lib/semantics/resolve-names.cc

```diff
@@ -404,13 +407,15 @@ class ModuleVisitor : public virtual ScopeHandler {
 
   void SetAccess(const parser::Name &, Attr);
   void ApplyDefaultAccess();
-  const Scope *FindModule(const SourceName &);
   void AddUse(const parser::Rename::Names &);
   void AddUse(const parser::Name &);
   // Record a use from useModuleScope_ of useName as localName. location is
   // where it occurred (either the module or the rename) for error reporting.
   void AddUse(const SourceName &location, const SourceName &localName,
       const SourceName &useName);
+  Symbol &BeginModule(const SourceName &, bool isSubmodule,
+      const std::optional<parser::ModuleSubprogramPart> &);
+  Scope *FindModule(const SourceName &, Scope * = nullptr);
```

Comment:
A name for that last optional argument would be informative.

Reply:
OK